UsbAccess/UsbAccess.cpp

```diff
@@ -763,5 +767,52 @@ namespace Plugin {
 
         return result;
     }
+
+    uint32_t UsbAccess::getUSBSizeWrapper(const JsonObject &parameters, JsonObject &response)
+    {
+        LOGINFOMETHOD();
+
+        bool result = false;
+        char cmd[128] = {'\0'};
+        char totalsize[20] = {'\0'};
+        char usedsize[20] = {'\0'};
+        char freesize[20] = {'\0'};
+
+        std::list<string> paths;
+        getMounted(paths);
+        if (!paths.empty()) {
+            snprintf(cmd, 127, "df -h | grep %s | awk \'{print $2 \" \" $3 \" \" $4}\'", (*paths.begin()).c_str());
+            FILE* fp = popen(cmd, "r");
+            if (fp) {
+                fscanf(fp,"%s %s %s", totalsize, usedsize, freesize);
+                response["totalsize"] = string(totalsize);
+                response["usedsize"] = string(usedsize);
+                response["freesize"] = string(freesize);
+                pclose(fp);
+                result = true;
+            }
+        }
+        returnResponse(result);
+    }
+
+    uint32_t UsbAccess::unmountUSBWrapper(const JsonObject &parameters, JsonObject &response)
+    {
+        LOGINFOMETHOD();
+
+        bool result = false;
+        char cmd[128] = {'\0'};
+
+        std::list<string> paths;
+        getMounted(paths);
+        if (!paths.empty()) {
+            snprintf(cmd, 127, "systemd-umount -u %s", (*paths.begin()).c_str());
```

Comment:
USBAccess now supports multiple USB Drives. Pls add support in this API also. It should take a path parameter like other APIs - https://rdkcentral.github.io/rdkservices/#/api/UsbAccessPlugin?id=getfilelist